src/sentry/api/bases/organization.py

```diff
@@ -516,7 +517,11 @@ def get_filter_params(
         return params
 
     def convert_args(
-        self, request: Request, organization_slug: str | None = None, *args: Any, **kwargs: Any
+        self,
+        request: Request,
+        organization_slug: str | int | None = None,
```

Comment:
Looks like you have to match the change in `RegionOrganizationIntegrationBaseEndpoint` for typing :O